src/main/webapp/css/style.css

```diff
@@ -1,3 +1,300 @@
 .favourite {
   color: yellow;
 }
+
+.jumbotron {
+    text-align: center;
+    background-color:#039BE5 !important;
+}
+
+/*.card {
+    text-align: center;
+    background-color:#2C384A !important;
+    width: 300px; 
+    height: 300px;
+    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 0.43);
+   
+}*/
```

Comment:
Please do not add commented code blocks. It is a better practice to just rewrite them later if required.